endpoints/api/namespacequota.py

```diff
@@ -154,9 +151,9 @@ def get(self, orgname, quota_id):
     def put(self, orgname, quota_id):
         orgperm = AdministerOrganizationPermission(orgname)
 
+        # Only super users can update quota
         if not features.SUPER_USERS or not SuperUserPermission().can():
-            if not orgperm.can():
-                raise Unauthorized()
+            raise Unauthorized()
```

Comment:
Superusers have their own endpoints for non-GET requests in `endpoints/api/superuser.py`
I might've missed making the ui's api calls in the previous PR though.

Reply:
So, do we want to make different API calls depending on the view it is called from?

Reply:
Not strictly necessary right away, since you've added the checks, just something to note, we should differentiate the calls from the superuser panel and org admin panel at some point. They're essentially the same thing, on different endpoints; it's easier to manage permissions.

Reply:
Fixed in the recent commit.